tests/lib/rules/no-shadow-route-definition.js

```diff
@@ -920,7 +1038,7 @@ describe('no-shadow-route-definition', () => {
     });
 
     expect(message).toStrictEqual(
-      'Route "second" (main/, 4L:10C) is shadowing route "first" (main/, 3L:10C)'
+      'Route "second" ("main/", 4L:10C) is shadowing route "first" ("main/", 3L:10C)'
```

Comment:
Do we need any other test cases for. the error message generation?

Reply:
I don't think so, functions generating this output have no decision points - https://github.com/ember-cli/eslint-plugin-ember/blob/8afa9ca42798a04256a451fc13551b1019a26f92/lib/rules/no-shadow-route-definition.js#L13 - so there's is no other path to test.